cli/package.json

```diff
@@ -61,5 +63,6 @@
         "description": "Modus Runtime Management"
       }
     }
-  }
+  },
+  "packageManager": "pnpm@8.14.1+sha512.856c4ecd97c5d3f30354ebc373cd32fb37274295c489db2d0f613a1e60f010cadfbb15036d525f3a06dec1897e4219b3c4f6d3be8f2f62fb0a366bee6aaa7533"
```

Comment:
I added a new debug target. Also added a note about using the debugger. I didn't like it though, because for interative commands like `modus new` I can't seem to input anything.

Reply:
My bad. I was using an automatic package manager switcher and defaulted to using PNPM.

Reply:
fixed lockfile